### PR TITLE
Feature/answers input

### PR DESCRIPTION
**Pull request by leviblit**

### CODE REVIEW

File: app/views/questions/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <%= f.label :lie %><br />
+    <%= text_area_tag("answers[][#{lie.id}][text]", lie.text) %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit('Update') %>
+  </div>
+<% end %>
+
+<%= link_to "Back", :back %>
```
Comment (blit-mattm) [inline app/views/questions/edit.html.erb]: i believe this needs a default fallback location incase back is undefined

File: app/views/questions/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    </div>
+  <% end %>
+
+  <h3>Your Lie</h3>
+
+  <p>
+    This is the answer that is a lie. Enter something that is false about you that people might think is true.
+  </p>
+
+  <% lie = @question.answers.detect(&:correct?) %>
```
Comment (blit-mattm) [inline app/views/questions/edit.html.erb]: this confused me at first, but the correct answer **is** the lie.

Comment (leviblit) [reply app/views/questions/edit.html.erb]: Yeah. I had to think about that for a while haha

File: app/controllers/questions_controller.rb [hunk 3]
```diff
@@ -1,2 +1,30 @@
 class QuestionsController < ApplicationController
+  before_action :prevent_unauthorized_access
+
+  def edit
+    question
+  end
+
+  def update
+    answers_params.each do |answers_params|
```
Comment (blit-mattm) [inline app/controllers/questions_controller.rb]: shadowed

File: app/views/questions/edit.html.erb (hunk 1 above)
Comment (leviblit) [reply app/views/questions/edit.html.erb]: Changed to root

File: app/controllers/questions_controller.rb (hunk 3 above)
Comment (leviblit) [reply app/controllers/questions_controller.rb]: Fixed.

**Pull request merged**